README.md

```diff
@@ -0,0 +1,19 @@
+# Server Duels
+
+## Description
+
+Server Duels is a Discord bot that allows you to duel your friends in turn-based combat.
+- [Wiki](https://github.com/domhankle/server-duels/wiki)
+- [License](server-duels/LICENSE.md)
```

Comment:
This is the right idea. The LICENSE.md file and README.md file are both at the root of the project though. So it should just be something like this:

`[License](LICENSE.md)`